libs/vkd3d/command.c

```diff
@@ -392,6 +397,15 @@ static HRESULT vkd3d_create_timeline_semaphore(struct d3d12_device *device, uint
     type_info.semaphoreType = VK_SEMAPHORE_TYPE_TIMELINE_KHR;
     type_info.initialValue = initial_value;
 
+    if (shared)
```

Comment:
This is only required for exporting semaphores, not import. Also somewhat missing a check for support with vkGetPhysicalDeviceExternalSemaphoreProperties.

Reply:
Actually, it might be relevant for import as well. I think you can CreateSharedHandle from an OpenSharedHandle.